CaliptraTrademarkAuditProcess.md

```diff
@@ -0,0 +1,152 @@
+  * Verify the self-assessment of the checklist and additional information provided by the manufacturer using the evaluation methodology.
+
+## Overview of Example Scheme
+
+For a formal certification scheme there are rules and documents of the scheme that defines it. In this example scheme definition, the discussed possible solution for the Caliptra Trademark Audit Process is commented within parenthesis.
+
+* Conformity Assessment Bodies (Entities)  
+  * Certification Body (CB).   
+    *(In this case the Caliptra TAC)*  
+  * Laboratories.   
+    *(In this case from OCP SAFE level 2 laboratories)*  
+* Related to Certification Body   
+  (In this case using and dependence to OCP SAFE)  
+  * Application to become Caliptra Trademark Laboratory   
+    *(In this case using and dependence to OCP SAFE)*  
+    * Criteria for Caliptra Trademark Laboratory  
+      * E.g. National accreditation needed, ISO 17025\. Note: ISO 17025 requires Evaluation Methodology of scheme to accredit laboratory, in general.  
+    * Technical capabilities for Caliptra  
+    * Terms and conditions  
+  * Audit process for Caliptra Trademark Laboratory (Technical capabilities)  
+    *(In this case using and dependence to OCP SAFE)*  
+  * Renewal of accepted Caliptra Trademark Laboratory  
+    *(In this case using and dependence to OCP SAFE)*  
+  * Publication of accepted Caliptra Trademark Laboratory  
+    *(In this case using and dependence to OCP SAFE)*  
+  * Audit protocols  
+    *(In this case using and dependence to OCP SAFE)*  
+* Related to Products and Caliptra Trademark Usage  
+  * Application for Caliptra Trademark Usage for a TOE  
+    *(Task for Caliptra TAC to develop.)*  
+  * Publication of all approved products for Caliptra Trademark usage  
+    *(Task for Caliptra TAC to develop.)*  
+  * Process specification for obtaining Caliptra Trademark  
+    *(A brief suggestion is outlined in this document)*  
+  * Rules for Caliptra Trademark Usage  
+    *(Task for Caliptra TAC to develop. See suggestions for some aspects in this document.)*  
+* Related to Evaluation  
+  * Evaluation methodology (audit process) (to be used by Laboratory or Vendor)  
+    *(Scope of current work)*  
+    * Checklist  
+    * How to apply checklist (Evaluation methodology)  
+  * *Evaluation Technical Report Template*  
+    *(Task for Caliptra TAC to develop or to delegate to the laboratories)*  
+  * *Communication between Laboratory and Manufacturer*  
+    *(Task for Caliptra TAC to develop or to delegate to the laboratories)*  
+  * *Communication between Laboratory and Caliptra Trademark Owner*  
+    *(Task for Caliptra TAC to develop.)*  
+  * *A forum where the Labs can ask questions and make sure the level of evaluation is more or less consistent across labs. In such a forum the scheme can be further developed and improved.*  
+    *(Task for Caliptra TAC to develop.)*
+
+## Specific Questions for the Scheme Rules
```

Comment:
Is there a need here to specify requirements for the impartiality of a lab? E.g. is it permissible that a lab supports the vendor in creating documentation? If yes, shouldn't it be specified that the lab team providing such "writing" services is separate from the evaluation team?